src/_P039_Thermosensors.ino

```diff
@@ -823,11 +840,12 @@ float readMax31855(struct EventStruct *event)
 
 
 
-  if((nullptr != P039_data)){
+  if(nullptr != P039_data){
 
       // FIXED: c.k.i. : moved static fault flag to instance data structure
       P039_data->sensorFault = false;
 
+      // chech for fault flags in LSB of 32 Bit messageBuffer - lowest
```

Comment:
"Check" ;)  (told you, entering the nitpicking phase :) )

Reply:
love it ! - this is indicating we are entering the last stage of your review...

Reply:
Solved in upcoming push !